assets/js/components/Work/Tabs/DownloadLinks.jsx

```diff
@@ -1,8 +1,12 @@
-import React from "react";
+import React, { useContext } from "react";
 import PropTypes from "prop-types";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { IIIFContext } from "../../IIIF/IIIFProvider";
+import { IIIF_SIZES } from "../../../services/global-vars";
+
+const WorkTabsDownloadLinks = ({ handleDownloadClick, filesetid }) => {
```

Comment:
@katdivyareddy10 Can we update `filesetid` to be `filesetId` in all occurrences?

Reply:
✅